### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by renatomefidf**
Ability to auto delete zips;
Travis upgrades

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
    - php vendor/bin/phpmd src/ text codesize,controversial,design,naming,unusedcode
    - php vendor/bin/phpunit
-after_script:
-   - php vendor/bin/coveralls
-   ## Scrutinizer
-   - wget https://scrutinizer-ci.com/ocular.phar
-   - php ocular.phar code-coverage:upload --format=php-clover ./build/logs/clover.xml
+
+after_script: >
+  php vendor/bin/coveralls &&
```
Comment (InFog) [inline .travis.yml]: Can you remove this line? I also have a PR to remove Coveralls :)

Comment (fabiorphp) [inline .travis.yml]: The PR to remove Coveralls was approved, please update your PR and remove duplicated commit messages.

**Comment by renatomefidf** [comment]
Hello @InFog and @fabiorphp 
I did a rebase for my PR! I don't know why scrutinizer failed, but it was executed: https://travis-ci.org/phansible/phansible/jobs/65088425#L306

**Comment by InFog** [comment]
Can you check that, @fabiorphp ?

**Comment by fabiorphp** [comment]
Done!

**Comment by naxhh** [comment]
We really need to thest phansible under all that versions? I think we should just launch our current versions and future ones only.

**Comment by InFog** [comment]
The current live env is 5.5, so I would go with >=5.5 and remove old ones.

**Comment by renatomefidf** [comment]
Just updated to >=5.5

The main feature of this PR is the auto deletion of the ZIP files, this will reduce some kinds of maintenance!

**Comment by InFog** [comment]
It looks ok now :) Thanks again, @renatomefidf!

I think @naxhh is the one to merge this.

**Pull request merged**